### ISSUE TITLE
Replace uses of _config with couchdb v2 compatible api

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#2856]
In particular, the user creation UI which calls _config to update admin passwords.

**Comment by garethbowen** [comment]
It's also used in data migrations.

**Comment by SCdF** [comment]
![image](https://cloud.githubusercontent.com/assets/583851/20483924/99dd01d0-afec-11e6-83d5-988d877cc844.png)
Sigh.

To be clear:
```
scdf at SCdF in ~
$ curl http://admin:pass@localhost:5984
{"couchdb":"Welcome","version":"2.0.0","vendor":{"name":"The Apache Software Foundation"}}
scdf at SCdF in ~
$ curl http://admin:pass@localhost:5984/_config
{"error":"not_found","reason":"Database does not exist."}
```

The documentation is just not up to date.

**Comment by SCdF** [comment]
APIs of interest:
 - ` /_membership` lists all members of the cluster
 - ` /_node/XXX/YYY` allows you to view the YYY endpoint of node XXX.

For example:
```
scdf at SCdF in ~
$ curl http://admin:pass@localhost:5984/_membership
{"all_nodes":["couchdb@localhost"],"cluster_nodes":["couchdb@localhost"]}
scdf at SCdF in ~
$ curl "http://admin:pass@localhost:5984/_node/couchdb@localhost/_config"
{"native_query_servers":{"query":"{mango_native_proc, start_link, []}"},"htt... etc
```

It very much doesn't make sense to me that admin credentials are per-node, but whatever. I guess the correct approach to make sure we support multiple nodes in the future would be to iterate over all nodes when we make changes, even if for now there is only ever one.

**Comment by garethbowen** [comment]
Is there no other/new API for modifying admin credentials? Maybe we should just disable changing the password for admins and make them go through [fu|faux]ton?

**Comment by SCdF** [comment]
Yeah I think that's the best approach, the code change required to do this is pretty horrid anyway.

It's also a good first step in us removing the use of CouchDB admins in medic-webapp (we should still have admins, they just shouldn't be the infallible gods that are official CouchDB admins).

**Comment by SCdF** [comment]
Removing usages of this endpoint in medic-webapp: https://github.com/medic/medic-webapp/pull/2943
Removing usages of this endpoint in medic-api: https://github.com/medic/medic-api/pull/106
Removing usages of this endpoint in medic-sentinel: https://github.com/medic/medic-sentinel/pull/87

**Comment by mandric** [comment]
Possibly related... we should be using /api/v1/users for any type of user/auth editing? https://github.com/medic/medic-webapp/issues/2048

**Comment by SCdF** [comment]
Yo @garethbowen we'll deffo need to discuss this one. Take a look at the PRs above and we'll have a chat about it.

**Comment by SCdF** [comment]
@mandric we could do, but putting the time in clearly is not important enough to anyone, since it got punted through 15 milestones without anyone working on it.

**Comment by garethbowen** [comment]
@SCdF Looks good. I'll leave the merge to you, or we can talk about it first.

**Comment by SCdF** [comment]
Merged to their 2.0 branches.

**Comment by garethbowen** [comment]
Will be ATed with CouchDB 2
